output.c

```diff
@@ -194,6 +195,18 @@ EXPORT void write_metainfo(FILE *f, struct metafile *m, unsigned char *hash_stri
 		fprintf(f, "6:source%lu:%s",
 			(unsigned long) strlen(m->source), m->source);
 
+	if (m->cross_seed) {
+		unsigned char rand_byte;
+		int value_length = 16;
```

Comment:
Wouldn't it be better to have `value_length` as a "proper" constant? (I'm thinking of `#define` or just `static const int`.)

Reply:
I agree. I added `#define CROSS_SEED_RAND_LENGTH 16` to `output.h`.